src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/ReduceTokensCodeCleanupProvider.vb

```diff
@@ -71,6 +71,28 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
                         If Not CaseInsensitiveComparison.Equals(valueText, idText) Then
                             Return newNode.ReplaceToken(literal, CreateDecimalLiteralToken(literal, valueText, value))
                         End If
+                    Case SyntaxKind.IntegerLiteralToken
+                        ' Get the literal identifier text which needs to be pretty listed.
+                        Dim idText = literal.GetIdentifierText()
+                        Dim value As Long = CType(literal.Value, Long)
+
+                        If value = 0 Then
+                            ' Overflow/underflow case or zero literal, skip pretty listing.
```

Comment:
Can we tell the difference? It'd seem odd to me that 001 would simplify but not 000.